vm/src/sysmodule.rs

```diff
@@ -222,6 +224,78 @@ fn sys_displayhook(obj: PyObjectRef, vm: &VirtualMachine) -> PyResult<()> {
     Ok(())
 }
 
+#[pystruct_sequence(name = "sys.getwindowsversion_type")]
+#[derive(Default, Debug)]
+#[cfg(windows)]
+struct WindowsVersion {
+    major: u32,
+    minor: u32,
+    build: u32,
+    platform: u32,
+    service_pack: String,
+    service_pack_major: u16,
+    service_pack_minor: u16,
+    suite_mask: u16,
+    product_type: u8,
+    platform_version: (u32, u32, u32),
+}
+
+#[cfg(windows)]
+fn sys_getwindowsversion(vm: &VirtualMachine) -> PyResult<PyTupleRef> {
```

Comment:
`PyResult<crate::obj::objtuple::PyTupleRef>` is also possible to avoid conditional `use` statement. But current state is also good.